azure-pipelines.yml

```diff
@@ -11,163 +11,127 @@ resources:
   repositories:
   - repository: cnp-azuredevops-libraries
     type: github
-    ref: master
+    ref: DTSPO-9663-template-file-cleanup
```

Comment:
please get this merged first

Reply:
Yeah, would be reverted before merged, was just a test

Reply:
@taherkhan30 ok to revert this now and do another review. `cnp` merged.